### PR TITLE
Added postgresdb resources and Created kubernetes page

### LINKED ISSUE #197: We Need Kubernetes Channel
**Is your feature request related to a problem? Please describe.**

- For Kubernetes related resources, Kubernetes channel needs to be created.

**Describe the solution you'd like**

- Add `Kubernetes `page to documentation section.

### PR DESCRIPTION
**Pull request by AbdullahOztuurkk**
## What type of PR is this? (check all applicable)

- [x] Added new link to category
- [x] New category added
- [ ] Fixed broken link
- [ ] Typo fix
- [ ] Readme update
- [ ] Optimization
- [ ] Documentation update
- [ ] Frontend update

## Description

- Resources for `postgresdb `has been added. Also has been created `kubernetes `page in documentation section.

## Related Issues & Documents

closes #197 

## QA Instructions, Screenshots, Recordings

image

### CODE REVIEW

**Comment by damla** [comment]
#14

**Comment by omerayyildiz** [comment]
Bir kedim bile yok. Anlıyor musun.

File: docs/databases/postgresdb.md [hunk 1]
```diff
@@ -8,4 +8,6 @@ sidebar_label: PostgresDB
 
 ## Intermediate
 
+- [PosgresSQL Course | AmigosCode](https://www.youtube.com/playlist?list=PLwvrYc43l1MxAEOI_KwGe8l42uJxMoKeS)
```
Comment (damla) [inline docs/databases/postgresdb.md:11]:
```suggestion
- [PostgresSQL Course | AmigosCode](https://www.youtube.com/playlist?list=PLwvrYc43l1MxAEOI_KwGe8l42uJxMoKeS)
```

**Review by damla — APPROVED** [review]
Thanks for contributing! 🥳

🚢🚢

**Review by damla — APPROVED** [review]

**Pull request merged**